tests/components/matter/test_light.py

```diff
@@ -240,7 +412,137 @@ async def test_turn_off(
 
     assert matter_client.send_device_command.call_count == 1
     assert matter_client.send_device_command.call_args == call(
-        node_id=light_node.node_id,
+        node_id=extended_color_light_node.node_id,
         endpoint=1,
         command=clusters.OnOff.Commands.Off(),
     )
+    matter_client.send_device_command.reset_mock()
+
+    # Turn the light on
```

Comment:
We don't need to duplicate the simple turn on call in the tests that should test color features.